### PR TITLE
chore, refactor(egen) : changes file extension from .json to .yaml, refactored code as per template guidelines

### PR DESCRIPTION
**Pull request by sumanvita-springml**
This PR includes the following updates:

1. Adds Colab Enterprise link and other structural changes from the new template.
2. Changes file extension from .json to .yaml in the cleanup section for deleting locally generated resources.
3. Changes REGION variable name to LOCATION.
5. Removes import os and IS_TESTING from the cleaning up section.
6. Adds comments for cells needed.
7. Removes the usage of future tense.
8. Replaces aip with aiplatform.
9. Contracts words like is not, do not, You are, etc.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/pipelines/control_flow_kfp.ipynb [hunk 1]
```diff
@@ -74,17 +76,16 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use the KFP SDK to build pipelines that use loops and conditionals, including nested examples.\n",
+        "In this tutorial, you learn how to use the KFP SDK to build pipelines that uses loops and conditionals, including nested examples.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/control_flow_kfp.ipynb:79]: parsing is not great. Suggested change: (I'm assuming "uses" agrees with "the KFP SDK" and not "pipelines, which is why you changed it from "use". But, it's very awkward as written above.)

- "In this tutorial, you learn how to use the KFP SDK, which uses loops and conditional including nested examples, to build pipelines."

File: notebooks/official/pipelines/control_flow_kfp.ipynb [hunk 2]
```diff
@@ -530,9 +473,9 @@
       "source": [
         "## Define a pipeline that uses control structures\n",
         "\n",
-        "The following example defines a pipeline that uses these components and demonstrates the use of  `dsl.Condition` and `dsl.ParallelFor`.\n",
+        "The following example, defines a pipeline component that demonstrates the use of `dsl.Condition` and `dsl.ParallelFor`.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/control_flow_kfp.ipynb:476]: remove the comma after "example". "The following example" is the subject of the sentence, not a parenthetical 👍 

- "The following example defines a pipeline component that..."

File: notebooks/official/pipelines/control_flow_kfp.ipynb [hunk 3]
```diff
@@ -670,11 +611,11 @@
         "\n",
         "<a href=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" target=\"_blank\"><img src=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" width=\"40%\"/></a> -->\n",
         "\n",
-        "In the Google Cloud Console, many of the pipeline DAG nodes expand or collapse when you click on them. Here's a partially-expanded view of the DAG (click image to see larger version).\n",
+        "In the Google Cloud Console, you can click on the pipeline DAG nodes to expand or collapse them. Here's a partially-expanded view of the DAG (click image to see larger version).\n",
         "\n",
         "<a href=\"https://storage.googleapis.com/amy-jo/images/mp/control_flow_dag.png\" target=\"_blank\"><img src=\"https://storage.googleapis.com/amy-jo/images/mp/control_flow_dag.png\" width=\"95%\"/></a>\n",
         "\n",
-        "You can see, for example, that the 'heads' condition passed, and thus the 'tails' condition— as we would expect— did not."
+        "You can see, for example, that the 'heads' condition passed, and thus the 'tails' condition failed as expected."
```
Comment (kittyabs) [inline notebooks/official/pipelines/control_flow_kfp.ipynb:618]: remove "thus"

**Review by kittyabs — COMMENTED** [review]
suggested edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**